docs/source/scheduling-policies.rst

```diff
@@ -130,11 +130,11 @@ functions in ``scheduler.py``.
 
 .. autosummary:: decide_worker
 
-.. autosummary:: Scheduler.decide_worker_non_rootish
+.. autosummary:: Scheduler._decide_worker_non_rootish
 
-.. autosummary:: Scheduler.decide_worker_rootish_queuing_disabled
+.. autosummary:: Scheduler._decide_worker_rootish_queuing_disabled
 
-.. autosummary:: Scheduler.decide_worker_rootish_queuing_enabled
+.. autosummary:: Scheduler._decide_worker_rootish_queuing_enabled
 
 .. autosummary:: Scheduler.worker_objective
```

Comment:
I think we should nuke this entire document. definitely the reference to the methods

Reply:
I don't care either way. If the interested user wants to read through thouse implementation details, sure, have at it.

Reply:
...though I do see the challenge of keeping this up-to-date (which I'm not sure it currently is, and frankly, I'm also not going to read it in full and compare it to the status quo today)

Reply:
Let's make this a follow-up issue in case someone else has strong opinions.

Reply:
#8467